components/edit/scriptNav.tsx

```diff
@@ -27,6 +28,12 @@ const ScriptNav: React.FC<ScriptNavProps> = ({ collapsed, setCollapsed }) => {
   const { script, loading } = useContext(EditContext);
   const { me } = useContext(AuthContext);
 
+  const handleNew = async () => {
+    createDefaultAssistant().then((script) => {
+      window.location.href = `/edit?id=${script?.id}`;
```

Comment:
It's usually better to navigate via the builtin nextjs router. Calling their methods performs some other functions behind the scenes in addition to setting `window.location.href`

For Client components: 
```ts
const router = useRouter() // import from `next/navigation` (not `next/router`)
// when you need to navigate
router.push("/location")
```

For Server components:
```ts
redirect("/url/path") // also import from `next/navigation`
```